R/ExtractData_Soils.R

```diff
@@ -819,6 +1022,13 @@ ExtractData_Soils <- function(exinfo, SFSW2_prj_meta, SFSW2_prj_inputs,
       fnames_in = SFSW2_prj_meta[["fnames_in"]], resume, verbose)
   }
 
+  if (exinfo$ExtractSoilDataFromISRICSoilGrid_Global_250m){
```

Comment:
our code style asks for a space between `){` in this case, see, e.g., https://style.tidyverse.org/syntax.html#spacing (2.2.2 Parentheses)